activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1041,17 +1041,11 @@ def foreign_key_options(from_table, to_table, options) # :nodoc:
         options
       end
 
-      def dump_schema_information #:nodoc:
-        versions = ActiveRecord::SchemaMigration.all_versions
-        insert_versions_sql(versions) if versions.any?
-      end
-
       def internal_string_options_for_primary_key # :nodoc:
         { primary_key: true }
       end
 
-      def assume_migrated_upto_version(version, migrations_paths)
-        migrations_paths = Array(migrations_paths)
+      def assume_migrated_upto_version(version)
```

Comment:
We need to deprecate before removing this argument. The method is on the public API http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-assume_migrated_upto_version